schemas/block.json

```diff
@@ -0,0 +1,257 @@
+							"enum": [
+								"attribute"
+							]
+						},
+						"attribute": {
+							"type": "string",
+							"description": "Use an attribute source to extract the value from an attribute in the markup. The attribute is specified by the attribute field, which must be supplied.\n\nExample: Extract the src attribute from an image found in the block’s markup."
+						}
+					},
+					"required": [
+						"type"
+					]
+				}
+			},
+			"additionalProperties": false
+		},
+		"supports": {
+			"type": "object",
+			"description": "It contains as set of options to control features used in the editor. See the the supports documentation at https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/ for more details.",
+			"properties": {
+				"anchor": {
+					"type": "boolean",
+					"description": "Anchors let you link directly to a specific block on a page. This property adds a field to define an id for the block and a button to copy the direct link.",
+					"default": false
+				},
+				"align": {
+					"default": false,
+					"description": "This property adds block controls which allow to change block’s alignment.",
+					"oneOf": [{
+							"type": "boolean"
+						},
+						{
+							"type": "array",
+							"items": {
+								"type": "string",
+								"enum": [
+									"wide",
+									"full",
+									"left",
+									"center",
+									"right"
+								]
+							}
+						}
+					]
+				},
+				"alignWide": {
+					"type": "boolean",
+					"description": "This property allows to enable wide alignment for your theme. To disable this behavior for a single block, set this flag to false.",
+					"default": true
+				},
+				"customClassName": {
+					"type": "boolean",
+					"description": "This property adds a field to define a custom className for the block’s wrapper.",
+					"default": true
+				},
+				"defaultStylePicker": {
+					"type": "boolean",
+					"description": "When the style picker is shown, a dropdown is displayed so the user can select a default style for this block type. If you prefer not to show the dropdown, set this property to false.",
+					"default": true
+				},
+				"html": {
+					"type": "boolean",
+					"description": "By default, a block’s markup can be edited individually. To disable this behavior, set html to false.",
+					"default": true
+				},
+				"inserter": {
+					"type": "boolean",
+					"description": "By default, all blocks will appear in the inserter. To hide a block so that it can only be inserted programmatically, set inserter to false.",
+					"default": true
+				},
+				"multiple": {
+					"type": "boolean",
+					"description": "A non-multiple block can be inserted into each post, one time only. For example, the built-in ‘More’ block cannot be inserted again if it already exists in the post being edited. A non-multiple block’s icon is automatically dimmed (unclickable) to prevent multiple instances.",
```

Comment:
```suggestion
					"description": "A non-multiple block can be inserted into each post, one time only. For example, the built-in 'More' block cannot be inserted again if it already exists in the post being edited. A non-multiple block’s icon is automatically dimmed (unclickable) to prevent multiple instances.",
```